pkg/apis/crd/v1alpha2/types.go

```diff
@@ -193,6 +195,8 @@ type PodOwner struct {
 	// Network interface name. Used when the IP is allocated for a secondary network interface
 	// of the Pod.
 	IFName string `json:"ifName,omitempty"`
+
+	MAC net.HardwareAddr `json:"mac,omitempty"`
```

Comment:
why does it need to be added to the `PodOwner` struct?

Reply:
no need to add, i removed it.